en/docs/manage-apis/design/api-security/oauth2/grant-types/jwt-grant.md

```diff
@@ -98,7 +93,7 @@ Now you have configured a service provider in WSO2 IS that can be used [later](#
 
 5. Navigate to the **Service Providers** section in the **Main** -> **Identity** section and click **Add.**
 
-6. Fill in the **Service Provider Name** and provide a brief **Description** of the service provider. See [Adding a Service Provider](https://is.docs.wso2.com/en/5.10.0/learn/adding-and-configuring-a-service-provider/) for more information.
+6. Fill in the **Service Provider Name** and provide a brief **Description** of the service provider. See [Adding a Service Provider](https://is.docs.wso2.com/en/6.1.0/guides/applications/register-sp/) for more information.
```

Comment:
@th33k as the instructions are for IS 7.x, we will need to point to the relevant documentation here. Also is it possible to check on adding IS 6.1.0 steps to some collapsible section for users who would be using IS 6.1.0? Then we will have to change the wording of the warning message at the start of this doc.